linetools/spectra/lsf.py

```diff
@@ -330,8 +333,24 @@ def load_STIS_data(self):
             data_aux = ascii.read(file_name, data_start=2, names=col_names)
             # reformat rel_pix
             data_aux['rel_pix'] = self.check_and_reformat_relpix(data_aux['rel_pix'])
+
+            # handle asymmetric STIS LSF
+            if data_aux['rel_pix'][len(data_aux['rel_pix']) // 2] == 0.:
+                pass
+            elif data_aux['rel_pix'][(len(data_aux['rel_pix']) // 2) - 1 ] ==0.:
+                data_aux.insert_row(0,data_aux[-1])
+                data_aux['rel_pix'][0]=-data_aux['rel_pix'][-1]
+            elif data_aux['rel_pix'][(len(data_aux['rel_pix']) // 2) + 1 ] ==0.:
+                data_aux.add_row(data_aux[0])
+                data_aux['rel_pix'][-1]=-data_aux['rel_pix'][0]
+
             # create column with absolute wavelength based on pixel scales
```

Comment:
I meant from 337-345. I agree it's a philosophical choice here, but I rather prefer to have a human being (developer) inspecting the LSF tabulated tables that will be ingested in linetools rather than having this piece of code that could easily fail (without raising an error) if the tables are given in the wrong format.